src/__tests__/extractMeta.js

```diff
@@ -518,15 +518,17 @@ describe('Metadata extractor', () => {
 
     it('Extracts metadata from component type', () => {
         const props = {
-            CustomComponent: ElementPropTypes.component("ComponentName")
+            CustomComponent: ElementPropTypes.component('ComponentName')
```

Comment:
We're using double quotes everywhere else in Element, should we do the same here? I wouldn't even ask, but I see you've included lint:fix, so it might be a quick change.